release/maven-central/heron-api.pom.template

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
'maven-central' changed to 'maven', including .sh script and release-process.md docs